frontend/src/pages/agreements/list/AgreementsFilterButton.module.css

```diff
@@ -0,0 +1,33 @@
+.filterButton {
+  background-color: #162E51;
```

Comment:
wonder if we have global colors we can use?

Reply:
I looked for it and didn't see it used elsewhere in the app.  In USWDS it isn't used other than for links where in this case it wouldn't semantically make sense.

Reply:
np, was hoping we had something like `bg-primary` or what have you.